AM: add information in Expression Language section about the access t… (#873)
gravitee-io/gravitee-docs: pages/am/3.x/user-guide/expression-language/user-guide-expression-language.adoc

NOTE: As you may have noticed, `claims` and `additionalInformation` contain the same data. Depending on the place where the Expression Language is defined, the `claims` attribute may not be accessible whereas the `additionalInformation` attribute will always be present.

> ==== Context Attributes

Reviewer: ```suggestion
==== Context attributes
```

> The context attributes will contains all the information you may add using Policies into the AM Flows. In addition of these information, you will find the `authFlow` attribute. This `authFlow` attribute is used to accumulate information accross the authentication flow thanks to the Enrich Authentication Flow Policy, it also contains the request parameters you defined using the link:https://datatracker.ietf.org/doc/html/rfc9126[PAR specification].  

Reviewer: ```suggestion
The Context attributes contain all the information you can add to AM Flows using policies. 
* You can use the `authFlow` attribute to accumulate information across the authentication flow thanks to the Enrich Authentication Flow Policy. 
* The `authFlow.requestParameters` attribute contains the request parameters defined using the link:https://datatracker.ietf.org/doc/html/rfc9126[PAR specification].  
```
